CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Changelog
 =========
 
+**debops.docker**
+
+`debops.docker master`_ - unreleased
```

Comment:
You can change this to plain `debops.docker master - unreleased` for now, Changelog will be updated later.